app/main.py

```diff
@@ -1,7 +1,25 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+
+from dotenv import load_dotenv
+load_dotenv()
+
+
+def get_weather() -> str:
+    response = requests.get(
+        url="https://api.weatherapi.com/v1/current.json",
+        params={"q": "Paris", "key": os.environ.get("API_KEY")}
```

Comment:
The URL should be defined as a constant in uppercase, as per the checklist requirement: "Use uppercase style for constants' names".